src/codebook/util/setDefaults.js

```diff
@@ -43,6 +59,25 @@ export function setDefaults(codebook) {
     }
   );
 
+  if (
+    codebook.config.variableLabels.length &&
+    defaultSettings.variableLabels.length
+  ) {
+    //merge the defaults with the user specified labels if both are populated
+    var userLabelVars = codebook.config.variableLabels.map(m => m.value_col);
+
+    //Keep the default label if the user hasn't specified a label for the column
+    defaultSettings.variableLabels.forEach(function(defaultLabel) {
+      if (userLabelVars.indexOf(defaultLabel.value_col) == -1) {
+        codebook.config.variableLabels.push(defaultLabel);
+      }
+    });
+  } else {
+    //otherwise just use whichever is populated or fallback to an empty object
+    codebook.config.variableLabels =
+      codebook.config.variableLabels || defaultSettings.variableLabels;
```

Comment:
hmm, so `[]||"test" == []", but `undefined||"test" == "test"` - didn't realize that. I guess we need to just write this out to be more verbose.